generated/nodejs/google-cloud-node/packages/monitoring/src/v3/metric_service_api.js

```diff
@@ -210,8 +207,7 @@ MetricServiceApi.prototype.monitoredResourceDescriptorPath = function monitoredR
  *   A fully-qualified path representing a monitored_resource_descriptor resources.
  * @returns {String} - A string representing the project.
  */
-MetricServiceApi.prototype.matchProjectFromMonitoredResourceDescriptorName =
-    function matchProjectFromMonitoredResourceDescriptorName(monitoredResourceDescriptorName) {
+MetricServiceApi.prototype.matchProjectFromMonitoredResourceDescriptorName = function(monitoredResourceDescriptorName) {
```

Comment:
I haven't looked at this closely but the generated method name still looks ridiculously long (that's unavoidable though).
Let's see how google-cloud-node people would say about it.